a3p-integration/proposals/b:replace-electorate/utils.js

```diff
@@ -0,0 +1,5 @@
+export const waitUntil = async waitTime => {
+  while (Math.floor(Date.now() / 1000) < waitTime) {
```

Comment:
Why wait repeatedly? Why not just compute the difference between waitTime and now and wait once?